tests/e2e/specs/drawer.cy.js

```diff
@@ -35,4 +38,14 @@ describe('Drawer component', () => {
       .get('#toggle-drawer')
       .should('be.visible')
   })
+  it('should drag to trigger resize', () => {
```

Comment:
Reloading the page is a good idea to prevent the previous `it` from polluting this one
```suggestion
  it('should drag to trigger resize', () => {
    cy.visit('/#/')
```